setup.py

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup
+
+setup(
+    name='ctlsettings',
+    version='0.1.0',
+    author='Columbia University Center for Teaching and Learning',
+    author_email='ctl-dev@columbia.edu',
+    url='https://github.com/ccnmtl/ctlsettings',
+    description='Columbia CTL common Django base settings',
+    long_description='common settings we use across all our projects',
+    install_requires = [
+        'django-debug-toolbar',
+        'coverage',
+        'django-smoketest',
+        'django-extensions',
+        'django-statsd-mozilla',
+        'sentry-sdk',
+        'django-storages',
+        'boto3',
+        'django-cacheds3storage',
+        'statsd',
+        'gunicorn',
+        'django-impersonate',
+    ],
+    scripts = [],
+    license = 'GPL-3.0-or-later',
+    platforms = ['any'],
+    zip_safe=False,
```

Comment:
I copied all these setup.py settings but many of them may no longer be necessary. I will take a look at the latest documentation, e.g.:
* https://packaging.python.org/en/latest/tutorials/packaging-projects/
* https://setuptools.pypa.io/en/latest/deprecated/zip_safe.html